Helper/Data.php

```diff
@@ -234,8 +234,8 @@ public function getRecurringTypes()
     public function getModes()
     {
         return [
-            '1' => 'Test Mode',
-            '0' => 'Production Mode'
+            '1' => 'Test',
+            '0' => 'Live'
```

Comment:
Do we have a ticket created to update docs with the new values?
For example here: https://docs.adyen.com/plugins/magento-2/set-up-the-plugin-in-magento#step-3-configure-the-plugin-in-magento ?

Reply:
Yes, https://youtrack.is.adyen.com/issue/DOC-3860 docs team has a whole new documentation lined up that reflects these changes